src/main/kotlin/com/beust/kobalt/maven/Pom.kt

```diff
@@ -62,9 +63,13 @@ public class Pom @javax.inject.Inject constructor(@Assisted val id: String,
         artifactId = XPATH.compile("/project/artifactId").evaluate(document)
         version = XPATH.compile("/project/version").evaluate(document)
         name = XPATH.compile("/project/name").evaluate(document)
-
-        var list = XPATH.compile("/project/properties").evaluate(document, XPathConstants.NODESET) as NodeList
+        var list = XPATH.compile("/project/repositories").evaluate(document, XPathConstants.NODESET) as NodeList
         var elem = list.item(0) as Element?
+        repos = elem.childElements()
+              .map({ it.getElementsByTagName("url").item(0).textContent })
+
+        list = XPATH.compile("/project/properties").evaluate(document, XPathConstants.NODESET) as NodeList
```

Comment:
Not a fan of reusing the same variable for two different XPaths, use a different one?